docs/styleguide.php

```diff
@@ -8,16 +8,14 @@
 <head>
 	<title><?php echo $DEFAULT_TITLE; ?> Style Guide</title>
 	<?php
-	include_once($SERVER_ROOT . '/includes/head.php');
-	include_once($SERVER_ROOT . '/includes/googleanalytics.php');
+		include_once($SERVER_ROOT . '/includes/head.php');
+		include_once($SERVER_ROOT . '/includes/googleanalytics.php');
+		include_once($SERVER_ROOT . '/includes/header.php');
```

Comment:
I moved this while I was troubleshooting. It doesn't seem to do any harm up here. What do you think?